givenergy_modbus/model/plant.py

```diff
@@ -96,9 +102,22 @@ def inverter(self) -> Inverter:
         return Inverter(self.register_caches[0x32])
 
     @property
-    def batteries(self) -> list[Battery]:
-        """Return Battery models for the Plant."""
-        return [
-            Battery(self.register_caches[i + 0x32])
-            for i in range(self.number_batteries)
-        ]
+    def batteries(self): # -> list[Battery]:
+        """Return LV Battery models for the Plant."""
+        if self.isHV:
+            return [
+                HVBattery.from_orm(self.register_caches[i + 0x50])
```

Comment:
Regression?  I think from_orm was a pydantic thing.

Reply:
Yep, this has been removed now